src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  value: [],
+};
+
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+
+  reducers: {
+    checkstatus: {},
+
+  },
```

Comment:
- **[OPTIONAL]** Your reducers should have actions to checkstatus, I mean, please 🙏🏽  complete the actions to successfully check the status.

## Hint
- Please try to understand how this code works:

```javascript
reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
```